src/Teams/Models/SingleSelectionQuestionModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Models
+{
+    public class SingleSelectionQuestionModel
+    {
+        public string Question { get; set; }
+        public List<string> Answers { get; set; }
```

Comment:
Use interfaces for non-primitive types whenever it possible.

Reply:
Even not IList, better to use IEnumerable or ICollection.